docs/jssdk.md

```diff
@@ -210,11 +217,11 @@ To create the 'My Account' menu for a logged in customer, you need the following
 
 ### Fetching Account Details
 
-By passing in the authorisation token, you can fetch all the account details using the `getAccountInfo` method.
```

Comment:
`getAccountInfo` is the method for getting the account details. The method should be called without any parameter.

Reply:
Done.